lib/fog/sakuracloud/models/volume/disk.rb

```diff
@@ -20,8 +20,12 @@ def delete
         alias_method :destroy, :delete
 
         def save
-          requires :name, :plan, :source_archive
-          data = service.create_disk(@attributes[:name], @attributes[:plan], @attributes[:source_archive]).body["Disk"]
+          requires :name, :plan
+          options = {
+            :source_archive=>@attributes[:source_archive],
+            :size_mb=>@attributes[:size_mb]
+          } 
+          data = service.create_disk(@attributes[:name], @attributes[:plan], options).body["Disk"]
```

Comment:
Ok. The this is something to be changed in another PR because it seems the whole codebase follow this pattern. :+1: